apps/settings/elements/root.html

```diff
@@ -121,6 +121,11 @@ <h2 data-l10n-id="personalization">Personalization</h2>
         <li class="themes-section" hidden>
           <a id="menuItem-themes" class="menu-item" data-icon="themes" href="#themes" data-l10n-id="themes">Themes</a>
         </li>
+        <li id="addons-section" hidden>
+          <a id="menuItem-addons" class="menu-item" data-icon="addons" href="#addons">
```

Comment:
nit: The current icon is oversized. Let's use "add" as the temporary icon. I'll file a followup bug requesting a new icon. Thanks.